lib/rage/cli.rb

```diff
@@ -25,6 +25,40 @@ def server
 
       ::Iodine.start
     end
+
+    desc 'routes', 'List all routes.'
+    def routes
+      # the result would be something like this:
+      # +----------------------+----------------------+----------------------+
+      # | Action               | Verb                 | Path                 | Controller#Action
+      # | index                | GET                  | /                    | application#index
+
+      # load config/application.rb
+      require File.expand_path('config/application.rb', Dir.pwd)
+      # load config/routes.rb
+      require File.expand_path('config/routes.rb', Dir.pwd)
+
+      routes = Rage.routes.router.routes
+
+      # construct a table
+      table = [['+---------------------------', '+---------------------------', '+---------------------------+']]
+
+      table << ['| Action'.ljust(20), '| Verb'.ljust(20), '| Path'.ljust(20), "| Controller#Action\n\n"]
```

Comment:
We should find the longest verb (that is always `DELETE`) and the longest path, and align based on those values, as there might be paths longer than 20 characters. I would suggest to use `sprintf` as it might work good here.